tests/jballerina-unit-test/src/test/resources/test-src/enums/enums.bal

```diff
@@ -115,13 +115,21 @@ function testEnumInRecursiveArray() {
     Keyword u = "union";
     assert(u, "union");
 
-    ApproxType b = "'boolean";
-    assert(b, 'boolean);
+    ApproxType b = "boolean";
+    assert(b, "boolean");
 
-    Keyword i = "'int";
-    assert(i, "'int");
+    Keyword i = "int";
+    assert(i, "int");
 }
 
+public enum CallState {
+    'new\u{5C}
+}
+
+function testEnumMemberContainingNumericEscape() {
+    CallState state = "new\\";
+    assert(state, "new\\");
```

Comment:
IMO we also need to test

```ballerina
    assert('new\u{5C}, "new\\");
```

Reply:
Yes, My intention was to test it but missed it when writing the test :-D